Cargo.toml

```diff
@@ -23,6 +10,26 @@ members = [
   "errors",
 ]
 
+[workspace.package]
+version = "0.9.0-dev"
```

Comment:
```suggestion
version = "0.10.0-dev"
```

Reply:
I am not sure if we should bump the version in this pull request, in my opinion we should do this in another pr?
If we do, we also need to update all the dependencies, because the version of the dependencies is not inherited at the moment (see conversation above).
I guess their is an GitHub-Action for that?